apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/HashValidatingDownloader.cs

```diff
@@ -27,13 +27,13 @@ namespace Google.Cloud.Storage.V1
     /// Subclass of <see cref="MediaDownloader"/> which validates the data it receives
     /// against a CRC32c hash set in the header.
     /// </summary>
-    internal sealed class HashValidatingDownloader : MediaDownloader
+    internal sealed class HashValidatingDownloader : ContentMetadataRecordingMediaDownloader
```

Comment:
I probably would have gone with composition here, but the classes are internal anyway so not important.

Reply:
Yes, I was wondering about that. I don't *like* the inheritance approach, but it's probably simpler than over-architecting. I definitely wouldn't have done it if it had been public :) (We can refactor if necessary in the future.)